### ISSUE TITLE
refactor require statements 

### ISSUE THREAD

**Issue by cg-cnu** [octokit/octokit.js#799]
The require statements are duplicated in couple of files. For example in `/lib/plugins/endpoint-methods/method.js` the require statements can be refactored 

from 

```js
const clone = require('lodash/clone')
const defaultsDeep = require('lodash/defaultsDeep')
const mapKeys = require('lodash/mapKeys')
```

to

```js
const {clone, defaultsDeep, mapKeys} = require('lodash')

```

I can help by creating a pr for fixing it. Let me know! Thanks 🙂

**Comment by gr2m** [comment]
the problem with this is that it would load the entire `lodash` package and hence increase the bundle size for the browser. The proper solution is to refactor the entire code to use es modules which supports proper tree shaking so unused code gets removed from bundles

**Comment by gr2m** [comment]
Let me know if you have any further questions about this :) I’ll close the issue as it’s not a bug or open feature request

**Comment by cg-cnu** [comment]
Hey @gr2m Again, thanks for being patient and explaining it to me. Now I understood the reason and started to notice the impact of it. I installed `import cost` for vscode and started observing my own repos as well. 

And you know what I did, removed an entire module from https://github.com/DAVFoundation/organization-contributors here... https://github.com/DAVFoundation/organization-contributors/pull/18

Thanks so much for your help! Cheers 🥂 😄
